packages/core/src/lib/core/types-for-lists.ts

```diff
@@ -108,19 +124,19 @@ function getIsEnabled(listsConfig: KeystoneConfig['lists']) {
         query: true,
         create: true,
         update: true,
-        delete: true,
-        filter: defaultIsFilterable ?? true,
-        orderBy: defaultIsOrderable ?? true,
+        delete: listConfig.kind === 'list',
+        filter: defaultIsFilterable ?? listConfig.kind === 'list',
+        orderBy: defaultIsOrderable ?? listConfig.kind === 'list',
```

Comment:
You are differentiating, but then not differentiating.  Either use `true` and ignore them, or set them to `false` for singletons (and then ignore them...)